python/turbodbc_test/test_select_arrow.py

```diff
@@ -72,6 +72,20 @@ def test_arrow_int_column(dsn, configuration):
 
 @for_each_database
 @pyarrow
+def test_arrow_int_column_adaptive(dsn, configuration):
+    with open_cursor(configuration) as cursor:
+        cursor.execute("SELECT 42 AS a")
+        result = cursor.fetchallarrow(adaptive_integers=True)
+        assert isinstance(result, pa.Table)
+        assert result.num_columns == 1
```

Comment:
I guess the only really valuable assertions are the `int8` type checks and the `42` result. Everything else is sufficiently tested elsewhere.